gmso/core/pairpotential_type.py

```diff
@@ -35,7 +35,7 @@ class PairPotentialType(ParametricPotential):
     def __init__(
         self,
         name="PairPotentialType",
-        expression="4 * eps * (sigma / r)**12 - (sigma / r)**6)",
+        expression="4 * eps * (sigma / r)**12 - (sigma / r)**6",
```

Comment:
I dont think the order of operations is correct here:

The LJ potential should be:
![this](https://quicklatex.com/cache3/80/ql_5a248c48f055a32918b31b83c05b6f80_l3.png)
```suggestion
        expression="4 * eps * ((sigma / r)**12 - (sigma / r)**6)",
```

Reply:
Thanks. Fixed.